tools/wptrunner/wptrunner/executors/executoratspi.py

```diff
@@ -4,18 +4,36 @@
 from gi.repository import Atspi
 import json
 import threading
+import time
 
-def find_active_tab(root):
+import sys
+
+
+def poll_for_active_tab(root, logger, product):
+    active_tab = find_active_tab(root, product)
+    iterations = 0
+    while not active_tab:
+        time.sleep(0.01)
+        active_tab = find_active_tab(root, product)
+        iterations += 1
+
+    logger.debug(f"Found ATSPI active tab in {iterations} iterations")
```

Comment:
```suggestion
    while not active_tab:
        time.sleep(0.01)
        active_tab = find_active_tab(root, product)

```

We can probably ditch these logs too (but keep the logger, that'll be handy going forward)